File: 设计模式/05单例模式/05单例模式/ViewController.m
//
//  ViewController.m
//  05单例模式
//
//  Created by Charlie on 2019/8/15.
//  Copyright © 2019 www.fgyong.cn. All rights reserved.
//

#import "ViewController.h"
#import "Singleton.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
	[super viewDidLoad];
	// Do any additional setup after loading the view.
	Singleton  *ton=[Singleton getInstace];
	//code
}


@end

File: Swift/FYJsonToModel/FYJsonToModel/ViewController.swift
//
//  ViewController.swift
//  FYJsonToModel
//
//  Created by Charlie on 2019/5/8.
//  Copyright © 2019 www.fgyong.cn. All rights reserved.
//

import Cocoa

class ViewController: NSViewController {

    @IBAction func valueChange(_ sender: NSPopUpButton) {
        print("str:\(String(describing: sender.selectedItem?.title))")
        sender.title = sender.selectedItem?.title ?? "Objec-C"
    }
    override func viewDidLoad() {
        super.viewDidLoad()

    }

    override var representedObject: Any? {
        didSet {
        }
    }


}

